webapp/README.md

```diff
@@ -6,7 +6,7 @@
 
 ## Getting Started
 
--   Make sure [Node.js](http://nodejs.org) is installed
+-   Make sure [Node.js v14.x](http://nodejs.org) is installed
```

Comment:
I originally tried running npm install with node 16 on my local environment, and met with the following [error](https://github.com/sass/node-sass/issues/2549). It appeared that for the node sass version specified in the package.json file would only be compatible with node 14, so I used that instead and managed to install the dependencies. I have updated the readme in the webapp section as well with the information, in case others might want to set it up locally as well.